src/shared/ui/date-start/ui/date-start.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import React, { HTMLProps } from 'react';
+import { COURSE_STALE_AFTER_DAYS } from '@/core/const';
+import { dayJS } from '@/shared/helpers/dayJS';
+import { getCourseDate } from '@/shared/helpers/getCourseDate';
+
+type DateStartProps = HTMLProps<HTMLTimeElement> & {
+  date: string;
+};
+
+export const DateStart = ({ date, ...props }: DateStartProps) => {
+  const freshDate = getCourseDate(date, COURSE_STALE_AFTER_DAYS);
```

Comment:
Do we really need this? If i understand correctly, it already comes here in the correct format (TBD or formatted date)